app/presenters/finder_presenter.rb

```diff
@@ -66,32 +65,6 @@ def page_metadata
     metadata.reject { |_, links| links.blank? }
   end
 
-  def results
-    @results ||= ResultSetParser.parse(
-      search_results.fetch("results"),
-      search_results.fetch("total")
-    )
-  end
-
-  def show_keyword_search?
```

Comment:
When would this have been false previously?

Reply:
I can't really see how this would ever have been false tbh. According to @sihugh it might have been for https://www.gov.uk/prepare-eu-exit/all.